### ISSUE TITLE
Checkbox component does not have a visible focus ring

### ISSUE THREAD

**Issue by lyzadanger** [hypothesis/frontend-shared#627]
When navigating by keyboard, the Checkbox component (both the legacy version and the updated, more relevant, variant) do not have a visible focus ring. Checkbox instances are focus-able, and respond to keyboard input to check/uncheck the underlying `input type="checkbox"` element, but there is no visible ring.
